### PR TITLE
LFX Summer 2024: Meshery

### PR DESCRIPTION
**Pull request by Yashsharma1911**

### CODE REVIEW

**Review by nate-double-u — COMMENTED** [review]
Hi @Yashsharma1911, thanks for this!

@leecalcote, @aabidsofi19, and @muzairs15 could you 👍 or approve this PR to confirm your participation?

@Yashsharma1911, Could you also resolve the merge conflicts?

**Comment by Yashsharma1911** [comment]
> Hi @Yashsharma1911, thanks for this!
> 
> @leecalcote, @aabidsofi19, and @MUzairS15 could you 👍 or approve this PR to confirm your participation?
> 
> @Yashsharma1911, Could you also resolve th emerge conflicts?

Thankyou @nate-double-u conflicts resolved now

**Review by MUzairS15 — APPROVED** [review]

**Review by leecalcote — APPROVED** [review]
LGTM

**Review by nate-double-u — APPROVED** [review]
Thanks everyone!

**Review by nate-double-u — CHANGES_REQUESTED** [review]
This should target `programs/lfx-mentorship/2024/02-Jun-Aug/project_ideas.md`, part of our process is moving from project ideas to the README as the LFX approves the project.

File: programs/lfx-mentorship/2024/02-Jun-Aug/README.md [hunk 1]
```diff
@@ -693,4 +693,4 @@ Mentee application instructions can be found on the [Program Guidelines](https:/
     - Michael Yuan (@juntao, michael@secondstate.io)
     - Hung-Ying Tai (@hydai, hydai@secondstate.io)
 - Upstream Issue: https://github.com/WasmEdge/WasmEdge/issues/3371
-- LFX URL: https://mentorship.lfx.linuxfoundation.org/project/d52d172e-598d-4817-be97-3338d5b96432
+- LFX URL: https://mentorship.lfx.linuxfoundation.org/project/d52d172e-598d-4817-be97-3338d5b96432
```
Comment (nate-double-u) [inline programs/lfx-mentorship/2024/02-Jun-Aug/README.md:696]: I'm not sure what this change is...

Comment (Yashsharma1911) [reply programs/lfx-mentorship/2024/02-Jun-Aug/README.md:696]: Probabaly a space change, thanks for pointing out, lemme remove this

Comment (nate-double-u) [reply programs/lfx-mentorship/2024/02-Jun-Aug/README.md:696]: thanks :)

Comment (Yashsharma1911) [reply programs/lfx-mentorship/2024/02-Jun-Aug/README.md]: Fixed

**Review by nate-double-u — APPROVED** [review]
Thanks for the updates!

/approve

**Pull request merged**